test/benchee/memory_measure_test.exs

```diff
@@ -0,0 +1,70 @@
+defmodule Benchee.MemoryMeasureTest do
+  # We cannot use async: true because of the test that we're running to ensure
+  # there aren't any leaked processes if functions fail while we're tracing
+  # them.
+  use ExUnit.Case
+  alias Benchee.MemoryMeasure
+  import ExUnit.CaptureIO
+
+  {otp_version, _} =
+    :otp_release
+    |> :erlang.system_info()
+    |> to_string
+    |> Integer.parse()
+
+  if otp_version > 18 do
```

Comment:
I think it might be better to do this using tags and configure from the `test_helper.exs`. For example, this whole module could use `@moduletag :memory_measurements` and then in `test_helper.exs` it could call:

```elixir
otp_release = List.to_integer(:erlang.system_info(:otp_release))
exclusions = if otp_release > 18, do: [], else: [memory_measure: true]

ExUnit.start(exclude: exclusions)
```

Reply:
Oh, yeah, I forgot to fix this all up a bit. Danke!